issue80-moved the table's data initiation into sql file

amirabramovich opened the pull request:
Bonus: added banner for application :)

<!-- Thanks for taking the time to write this Pull Request ❤️ -->

## 🚀 Description

<!-- Describe your changes in detail -->

## 📄 Motivation and Context

<!-- Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->

## 🧪 How Has This Been Tested?

<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->

## 📷 Screenshots (if appropriate)

<!-- Please provide a screenshot of your change -->

## 📦 Types of changes

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] My code follows the code style of this project(Do your best to follow code styles. If none apply just skip this).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Commit:
added banner for application and moved the table's data initiation into sql file

DDmit04 commented:
will it be possible to use current [postman configuration](https://github.com/jmprathab/MyHome/tree/master/postman) after this change? Environment varibles in this config are very comfortable for testing

jmprathab commented on src/main/resources/application.yml:
```diff
@@ -25,7 +25,9 @@ server:
-  application.name: "myhome-service"
+  application:
+    name: "myhome-service"
+    version: 1.2.0-release
```
Let's not include this in the banner. As this gets outdated very soon, also we should take care of updating this after every release.
I am wondering if there is any way to automatically get it from Gradle configuration.

jmprathab commented on src/main/java/com/myhome/bootstrap/DataLoader.java:
```diff
@@ -59,7 +50,6 @@
 
   @Override public void run(String... args) throws Exception {
     //loadPostmanData();
```
Like @DDmit04 mentioned. We have to look for a way to include this in the `data.sql` file.

amirabramovich replied:
please create new task for it

Commit:
fixed banner to display application name + version from gradle, merged test classes to improve performance

amirabramovich commented:
hi,
i fixed the banner, now it takes information from gradle. for some reason the application name & version are visable only by running with java -jar (please check the first answer [here](https://stackoverflow.com/questions/34519759/application-version-does-not-show-up-in-spring-boot-banner-txt)).
also i fixed the JUnits, issue was with the inserts order in data.sql , and also it seems that every integration test class loads data.sql before the tests so i temporarily merged the two test classes into single class (need to figure out later how to clean data before test class and implement it, maybe should create issue for it).
also regarding the postman changes, please create a separate issue for it and describe there exactly what is the purpose of postman initiation . 
regarding the issue which i took #80  - i pretty sure that task is completed here so we can merge and close the issue (and open few new tasks for the new issues).

mslowiak commented:
@amirabramovich `DirtyContext` annotation can help with that on this moment
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/annotation/DirtiesContext.html

amirabramovich commented:
@mslowiak
Tried that, didn't work for some reason :(

jmprathab commented:
> @mslowiak
> Tried that, didn't work for some reason :(

Did you try it in conjunction with `@Transactional` ?

jmprathab reviewed (approved):
Looks good.
